packages/react/src/components/form/initial/header.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { useConfiguration } from "../../../hooks/use-configuration";
+import { TextConfig } from "../../text/constants";
+import { Text } from "../../text";
+
+import * as styles from "./initial.css";
+
+type Props = {
+  text: TextConfig;
+};
+export const HeaderSlot = ({
+  children,
+}: {
+  children?: React.ReactNode | ((props: Props) => React.ReactNode);
+}) => {
+  const { text } = useConfiguration();
+
+  const Heading = React.useMemo(() => {
+    return (
+      <div className={styles.header}>
+        <Text
+          as="h1"
+          variant={{ size: "2xl-title", weight: "bold" }}
+          t="initial.title"
+        />
+        <Text variant={{ color: "tertiary" }} as="h2" t="initial.subtitle" />
```

Comment:
@ikovic Since the components were broken down, there is a change I made in this [PR](https://github.com/slashid/javascript/pull/156) which won't be reflected here, so could you please change the `variant` to this: `variant={{ color: "contrast", weight: "semibold" }}`